File: client/src/mapposition/location.ts
import {Importance} from "./importance";

export class Location{
  longitude: number = 15.887;
  latitude: number = 50.054;
  specialization: string = 'Diabetologia';

  constructor(longitude: number, latitude: number, specialization: string) {
    this.longitude = longitude;
    this.latitude = latitude;
    this.specialization = specialization;
  }
}

File: src/main/java/com/example/wizytydomowe/Doctor/Location.java
package com.example.wizytydomowe.Doctor;

import com.example.wizytydomowe.Appointment.Importance;
import lombok.Data;
import lombok.RequiredArgsConstructor;

@Data
@RequiredArgsConstructor
public class Location {
    private double longitude;
    private double latitude;
    private Importance importance;
    private String specialization;
}
